README.md

```diff
@@ -37,8 +38,10 @@ On the new component panel, copy and paste the following attribute template into
 }
 ```
 
+Remove the "classifications" or "objects" section depending on if your ML model is a classifier or detector.
+
 > [!NOTE]  
-> For more information, see [Configure a Robot](https://docs.viam.com/manage/configuration/).
+> For more information, see [Configure a Machine](https://docs.viam.com/configure/).
 
 ### Attributes
 
```

Comment:
Can you update line 48 to remove base? No need to replace with "camera" since the model name already has camera in it.